docs/Makefile

```diff
@@ -19,7 +19,12 @@ docs-debug: docs docs-image
 		--rm \
 		--name vmdocs-docker-container \
 		-p 1313:1313 \
-		-v ./docs:/opt/docs/content/victoriametrics vmdocs-docker-package
+		-v ./docs/anomaly-detection:/opt/docs/content/victoriametrics \
```

Comment:
```suggestion
		-v ./docs/anomaly-detection:/opt/docs/content/anomaly-detection \
```

Reply:
updated target to mount all directories in docs